src/App.tsx

```diff
@@ -15,71 +16,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  }
+  clearState = () => {
+    this.setState({ selectedGood: ''})
+  }
+  changeState = (good: string) => {
+    this.setState({ selectedGood: good})
+  }
+  render() {
+    const { selectedGood } = this.state;
+    return (
+      <main className="section container">
+        {selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={this.clearState}
```

Comment:
here you can add comment `// eslint-disable-next-line jsx-a11y/control-has-associated-label`